lib/learn_test/runner.rb

```diff
@@ -57,7 +57,8 @@ def strategies
         LearnTest::Strategies::Protractor,
         LearnTest::Strategies::JavaJunit,
         LearnTest::Strategies::Mocha,
-        LearnTest::Strategies::PythonUnittest,
+        # LearnTest::Strategies::PythonUnittest, # removed as curriculum switched entirely to Pytest and test file naming convention may intersect with pytest test file naming convention
```

Comment:
Is there a reason to keep this `PythonUnittest` stuff hanging around, or can we just rm it all and trust that anyone looking to re-implement down the line can find it in the Git history? 🗑 ❓

Reply:
I am wondering the same -- see main comment thread

Reply:
Word; sorry for missing that earlier!

We should figure out whether `"we are deprecating it from curriculum"` means "we can immediately stop supporting it" or "we aren't going to write any new material that leverages it but we need to support existing material for X amount of time"

If it's the former, my preference is for full removal of the entire now-dead code path

Reply:
working on getting 100% confirmation on this now